### ISSUE TITLE
Save button sometimes does not have keyboard focus after using copy button

### ISSUE THREAD

**Issue by IanN-GH** [mozilla/pontoon#2604]
I had about 90+ strings to localise, so clicked on the link for missing on the dashboard.
STR
1/ Click on the first entry.
2/ Wait to see if it pre-fills, if not then click on the copy button.
3/ Either way then check the entry is correct, press the Enter key.

Expected result
1/ Save button is activated

Actual result
1/ Most of the time the save button is activate but occasionally it wasn't. I did notice it happened on entries with 3 fields, label, shortcut and tooltip - though couldn't say if happened to all of those or just some of those. When it did happen, the cursor caret was flashing at the end of the first field

**Comment by IanN-GH** [comment]
Not seen it happen recently, I will keep checking each time more strings need localising.

**Comment by mathjazz** [comment]
Thanks. I'll close the issue for now. Please reopen if you bump into the problem again.
